test/__tests__/test.re

```diff
@@ -16,6 +16,7 @@ open Belt.Result; */
 [@decco] type l('a) = list('a);
 [@decco] type o('a) = option('a);
 [@decco] type r('v, 'e) = Belt.Result.t('v, 'e);
+[@decco] type d('v) = Js.Dict.t('v);
```

Comment:
Could you write tests that use this type? Or [give me permission to push](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)?